frontend/static/messages/nl.json

```
@@ -3287,10 +3294,18 @@
         "actionPush": "Pushmelding",
         "optionsSection": "Opties",
         "cooldownMinutes": "Wachttijd (minuten)",
+        "cooldownSeconds": "Wachttijd (seconden)",
```

Biome (2.4.4)

[error] The key cooldownSeconds was already declared.

(lint/suspicious/noDuplicateObjectKeys)